action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'GitHub Forks Sync Action'
+description: 'GitHub Action to synchronise forks'
+author: 'TobKed'
+branding:
+  icon: upload-cloud
+  color: green
+inputs:
+  github_token:
+    description: 'Token for the repo. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
+    required: true
+  upstream_repository:
+    description: 'Repository name to clone.'
+    required: true
+  target_repository:
```

Comment:
What do you think about support for a branch other than the master? Master is now an offensive word.
https://github.com/apache/airflow/issues/9351

Reply:
I will add support for other branch names in later PRs.